Research Papers/Shallow learning/README.md

```diff
@@ -168,15 +185,28 @@ Using Utility Graphs </a>by Valentin Robu, D.J.A. Somefun, J.A. La Poutre. AAMAS
 
+<details> <summary> <a href="https://nscpolteksby.ac.id/ebook/files/Ebook/Computer%20Engineering/Algorithmic%20Game%20Theory%20%282007%29/22.%20Chapter%2021%20-%20The%20Price%20of%20Anarchy%20and%20the%20Design%20of%20Scalable%20Resource%20Allocation%20Mechanisms.pdf"> The Price of Anarchy and the Design of Scalable Resource Allocation Mechanisms </a>by Johari R. Algorithmic Game Theory, 2007. <a href="link">  </a> </summary> In this chapter, we study the allocation of a single infinitely divisible resource among multiple competing users. While we aim for efficient allocation of the resource, the task is complicated by the fact that users’ utility functions are typically unknown to the resource manager. We study the design of resource allocation mechanisms that are approximately efficient (i.e., have a low price of anarchy), with low communication requirements (i.e., the strategy spaces of users are low dimensional). Our main results concern the proportional allocation mechanism, for which a tight bound on the price of anarchy can be provided. We also show that in a wide range of market mechanisms that use a single market-clearing price, the proportional allocation mechanism minimizes the price of anarchy. Finally, we relax the assumption of a single market-clearing price, and show that by extending the class of Vickrey–Clarke–Groves mechanisms all Nash equilibria can be guaranteed to be fully efficient. <br> - </details>
```

Comment:
Also ecos? There is also the applied category and we could start a subcategory for resource allocation if we find more papers like this.